### PR TITLE
Add a tip to an example

### PR DESCRIPTION
**Pull request by abdounikarim**
Hello,

Can we add this tip ? The link is on the video but not on the text. 

Cheers 😉

### CODE REVIEW

File: sfcasts/ep2-security/production-docs.md [hunk 1]
```diff
@@ -143,5 +143,9 @@ you *could* always create an event subscriber on the `kernel.request` event -
 now called the `RequestEvent` in Symfony 4.3 - and, if the URL is `/api` *and*
 the request format is HTML, return a 404.
 
+***TIP
+You can see an example here : [https://bit.ly/2YmR6Uh](https://bit.ly/2YmR6Uh)
```
Comment (bocharsky-bw) [inline sfcasts/ep2-security/production-docs.md]: Can we put `:` close to `here`, i.e. `here: [https://bit.ly/2YmR6Uh](https://bit.ly/2YmR6Uh)`? :p

**Review by bocharsky-bw — APPROVED** [review]
I like this idea in general 👍

File: sfcasts/ep2-security/production-docs.md (hunk 1 above)
Comment (abdounikarim) [reply sfcasts/ep2-security/production-docs.md]: @bocharsky-bw grrr, english syntax 😠 is it good for you now ?

Comment (bocharsky-bw) [reply sfcasts/ep2-security/production-docs.md]: Yes, thank you!

**Comment by weaverryan** [comment]
Thanks @abdounikarim!

**Pull request merged**